### PR TITLE
Add TSCH timeslot element description

### PR DESCRIPTION
**Pull request by atiselsts**
A documentation improvement. This is based on the initial work by Rajeev Pinare.

### CODE REVIEW

File: os/net/mac/tsch/tsch-timeslot-timing.c [hunk 1]
```diff
@@ -47,7 +47,21 @@
...
+ * RxOffset    -> beginning of the timeslot to when the receiver shall be listening
+ * RxAckDelay  -> end of frame to when the transmitter shall listen for ACK
+ * TxAckDelay  -> end of frame to the start of ACK tx
+ * RxWait      -> time to wait for start of frame (Guard time)
+ * AckWait     -> min time to wait for start of an ACK frame
+ * RxTx        -> receive-to-transmit switch time (NOT USED)
+ * MaxAck      -> TX time to send a max length ACK
+ * MaxTx       -> TX time to send the max length frame
+ *
+ * The default timeslot timing in the standard is a guard time of
```
Comment (yatch) [inline os/net/mac/tsch/tsch-timeslot-timing.c]: It'd be nice to have a reference to IEEE802.15.4-2015, something like "The timeslot structure in Figure 6-30 of IEEE 802.15.4-2015 is assumed, although some of the time periods have different names."?

Comment (atiselsts) [reply os/net/mac/tsch/tsch-timeslot-timing.c]: I added a note about the figure in the standard, thanks!

Comment (yatch) [reply os/net/mac/tsch/tsch-timeslot-timing.c]: 👍

**Comment by rajeev1986** [comment]
@atiselsts and @yatch. Thanks for incorporating the changes.

**Pull request merged**